### PR TITLE
fix(kit): `InputTag` fix pasting behaviour

### LINKED ISSUE #2358: 🐞 - unexpected paste behavior in `InputTag`
### Which @taiga-ui/* package(s) are the source of the bug?

kit

### Please provide a link to a minimal reproduction of the bug

https://stackblitz.com/edit/taiga-starter

### Is this issue blocking you?

Non-Blocking

### Description

When you set `tuiTextfieldMaxLength`, its limited tag length while typing. But when pasting text with many tags separated by commas, the behavior is unexpected - `tuiTextfieldMaxLength` does not work for each tag individually, but for the entire inserted string, and as a result, even though each tag fits in length, the string is cutoff.

### Angular version

latest

### Taiga UI version

latest

### Which browsers have you used?

- [X] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### Which operating systems have you used?

- [ ] macOS
- [X] Windows
- [ ] Linux
- [ ] iOS
- [ ] Android

### PR DESCRIPTION
**Pull request by vladimirpotekhin**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [ ] Documentation content changes

## What is the current behavior?

Closes #2358

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by splincode — APPROVED** [review]

File: projects/demo/src/modules/components/input-tag/input-tag.template.html [hunk 1]
```diff
@@ -141,6 +142,17 @@
...
                 when item from list selected)
             </ng-template>
+            <ng-template
+                i18n
+                documentationPropertyName="maxLength"
+                documentationPropertyMode="input"
+                documentationPropertyType="number"
+                [(documentationPropertyValue)]="maxLength"
+            >
+                Textfield value to subscribe on input or setting it from the outside (emits
```
Comment (waterplea) [inline projects/demo/src/modules/components/input-tag/input-tag.template.html:152]: Text is wrong

File: projects/kit/components/input-tag/input-tag.template.html [hunk 2]
```diff
@@ -100,6 +100,7 @@
                                 [tuiFocusable]="computedFocusable"
                                 [ngModel]="search"
                                 (ngModelChange)="onInput($event)"
+                                (paste.prevent)="onPaste($event)"
```
Comment (waterplea) [inline projects/kit/components/input-tag/input-tag.template.html:103]: Technically we could also handle `drop` the same way. Could you give it a spin?

**Review by waterplea — APPROVED** [review]

File: projects/demo/src/modules/components/input-tag/input-tag.template.html (hunk 1 above)
Comment (vladimirpotekhin) [reply projects/demo/src/modules/components/input-tag/input-tag.template.html]: thanks

File: projects/kit/components/input-tag/input-tag.template.html (hunk 2 above)
Comment (vladimirpotekhin) [reply projects/kit/components/input-tag/input-tag.template.html:103]: done ✅

**Pull request merged**